api/organization_members.go

```diff
@@ -495,14 +494,14 @@ func (a *API) updateOrganizationMemberRoleHandler(w http.ResponseWriter, r *http
 		errors.ErrInvalidUserData.Withf("invalid role").Write(w)
 		return
 	}
-	if err := a.db.UpdateOrganizationMemberRole(org.Address, uint64(userIDInt), db.UserRole(update.Role)); err != nil {
-		errors.ErrInvalidUserData.Withf("member not found: %v", err).Write(w)
+	if err := a.db.UpdateOrganizationUserRole(org.Address, uint64(userIDInt), db.UserRole(update.Role)); err != nil {
+		errors.ErrInvalidUserData.Withf("user not found: %v", err).Write(w)
 		return
 	}
 	apicommon.HTTPWriteOK(w)
 }
 
-// removeOrganizationMemberHandler godoc
+// removeOrganizationUserHandler godoc
 //
 //	@Summary		Remove a user from the organization members
```

Comment:
This line and the next one still refer to members

Reply:
:100: ,fixed